src/text-editor.coffee

```diff
@@ -1310,14 +1351,27 @@ class TextEditor extends Model
   #   * `position` (optional) Only applicable to decorations of type `overlay`,
   #     controls where the overlay view is positioned relative to the marker.
   #     Values can be `'head'` (the default), or `'tail'`.
+  #   * `name` (optional) Only applicable to decorations of type `gutter`. Specifies
+  #      which gutter this applies to. If missing, the default gutter is assumed.
   #
   # Returns a {Decoration} object
   decorateMarker: (marker, decorationParams) ->
     if decorationParams.type is 'gutter'
+      # #TODO REVIEW: should we keep 'gutter', now that there can be many gutters?
```

Comment:
We should pass an additional option to suppress the deprecation warning when it's being used internally by the `GutterContainer`.